internal/repository/task_repository.go

```diff
@@ -8,14 +8,13 @@ import (
 	"github.com/uptrace/bun"
```

Comment:
You Don't need to call it "task_repository" you're already in the repository folder "task" is fine.
Also, add user repo and user handler and add an endpoint that gets all the tasks for a user.